recipe/meta.yaml

```
@@ -127,7 +127,7 @@ outputs:
         - {{ pin_subpackage('libmambapy', exact=True) }}
       run:
         - python
-        - conda >=23.9,<24
+        - conda >=23.11,<24
```

Given the comments from, e.g., https://github.com/mamba-org/mamba/issues/3033#issuecomment-1840688624 and https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/608#discussion_r1415455300, it would make sense to have this as
```suggestion
        - conda >=23.11,<23.12
```
. Whenever there is a new `conda` release for which `mamba` is still compatible with, we can just relax the constraint and issue a new build here.